ivadomed/losses.py

```diff
@@ -51,16 +51,19 @@ class DiceLoss(nn.Module):
     Attributes:
         smooth (float): Value to avoid division by zero when images and predictions are empty.
```

Comment:
```suggestion
        smooth (float): Value to avoid division by zero when images and predictions are empty.
        sample_wise (bool): If set, dice loss is averaged over sample instead of over batch
```